scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -242,6 +243,55 @@
 
   (restore-offers [this pool-name offers])
 
+  (autoscaling? [_]
```

Comment:
Shouldn't this be checked in the factory? If someone puts in a negative number, they're doing something really really weird. That way you just check it once, and return true/false here.

Reply:
Will do, thanks

Reply:
Moved this to the factory, thanks